executor/app.py

```
@@ -210,6 +214,92 @@ def _initialize_sandbox_claude(auth_token: str, task_id: str) -> None:
+            # Save file to attachments directory
+            file_path = os.path.join(attachments_dir, att.original_filename)
+            with open(file_path, "wb") as f:
+                for chunk in response.iter_content(chunk_size=8192):
+                    f.write(chunk)
+
+            logger.info(
+                f"[SandboxInit] Downloaded attachment '{att.original_filename}' to {file_path}"
+            )
+            success_count += 1
+
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)